permission_prompt/explainer.md

```diff
@@ -136,16 +135,31 @@ Alternatively, the permission only lasts shortly for a certain origin, similarly
 
 ### Automatically deny similar requests
 
-To avoid the abuse of the permission prompt, we might also want some automatically deny operation for similar requests. Here we propose some alternatives:
+To prevent abuse of the permission prompt, we might also want to automatically deny permission in some cases. Here we propose some alternatives:
 
 * Show `cancel` and `allow` instead of `block` and `allow` on the first time the prompt shows and add a “Do not show this again.” check box on the second prompt.
 * Automatically deny the request to the same local device in the current document.
 * Automatically deny the request to the same local device for the specific public website for a while.
 
-## Security and privacy considerations
+### IDs
 
-See also [`./security_privacy_self_review.md`](./security_privacy_self_review.md)
+The preflight request sent with plaintext HTTP can be hijacked which means that the `-ID` provided by the local device can be leaked to third-party attackers. However, this also means that the attacker is already in the private network. In this case, it doesn't matter whether the public page is allowed to make requests on the private network  since the attacker can make the request themselves.
 
-### IDs
+### Behavior of `targetAddressSpace` and fetch
 
-The preflight request sent with plaintext HTTP can be hijacked which means that the `-id` provided by the local device can be leaked to third-party attackers. However, this also means that the attacker is already in the private network. In this case, it doesn't matter whether the public page is allowed to make requests on the private network  since the attacker can make the request themselves.
+It might be annoying on one hand and limited developers ability to request private network resource to fetch only on
+the other if `targetAddressSpace` as an fetch option required to be set by developers manually every time. In this
+case, we once agreed on preserving the permission for the document's entire lifetime if once granted via fetch, and
+the document can perform arbitrary fetches with media tags, XHRs, etc. However, such behavior would still cause
+problems such as the DNS could be rebind in the middle of a document's lifetime.
```

Comment:
```suggestion
Requiring every single request to the private network to be issued with `fetch()` and the new `targetAddressSpace` might not support all developer use cases. We could improve this by only requiring it for the first request to the target origin. After a successful `fetch()` with `targetAddressSpace`, the browser could store a mapping from the target origin to the given `targetAddressSpace` for the remainder of the lifetime of the execution context. The execution context could then perform arbitrary fetches with media tags, XHRs, etc to the tartget origin.

Note that this might cause problems if the DNS entry for the target domain changes to point to a different IP address space in the middle of an execution context's lifetime.
```